meetings/2021-03-22.md

```diff
@@ -0,0 +1,91 @@
+Sarven: If we think it can be used beyond solid we could consider registering with IANA.
+Henry: Moving from "WebID" to something more general such as "Solid" would be good if one had something more general, allowing one to try out different options such as use of Credentials. Just changing the name though does not give a more general protocol. To get that one would need to add some parameters and values to allow the client to know the difference. Without those features the change brings a semblance of generality without providing one. And one would need to have implementations make a change without a benefit to come with it.
+...: I think JSON folks may at an IRI - in a position where a short attributes would have done just as well - as something odd or even weird.
+Aaron: Those JWTs get sent with every authenticated request. Short claims like `iat` etc. allow more compact tokents. If we change it to IRI it takes more bytes.
+Pavlik: scopes have ofte IRI as values. Also we had proposal to exchange bigger token for short token used on each request, this would also go along UMA2.0 approach with delegating claims gathering.
+Aaron: Let's continue in issue.
+Henry: It's really up to those implementing this to decide. I have not implemented the OIDC connect protocol, so please take my statements with a grain of salt there.
+
+
+### [specify parameters in WWW-Authenticate header #152](https://github.com/solid/authentication-panel/issues/152)
+
+Pavlik: ...
+Aaron: I think it is important to have something in WWW-Authenticate header. Is the question what the structure of the fields are?
+Henry: WWW-Authenticate has standard on what you can put in that response header
+...: HTTP-Sig has also uses that header, and implementing that I found some new IETF spec giving advice on how to structure
+those headers:
+Sarven: https://tools.ietf.org/html/rfc8941
+Henry: https://www.fastly.com/blog/improve-http-structured-headers
+
+### [Solid OIDC Conformance Classes #133](https://github.com/solid/authentication-panel/issues/133)
+
+Sarven: We were discussing documenting implementation, we tried to identify what kinds
+of implementation there are. Spec should indicate for each class what it needs to implement to conform.
+Pavlik: I could take action to add that section to the spec. Basing it 
+
+
+### [Clarifying specific areas of overlap with AuthZ #145](https://github.com/solid/authentication-panel/issues/145)
+
+Aaron: if RS needs to provide link to AS, how this would work with solid
+Pavlik: I mean AS associated with RS, distinct from IdP associated with user
+Benoît: As implementation its sometimes hard which pannel is responsible for which part
+Clarifying the overlap would help on implementation side
+Henry: In the HttpSig approach, there is a simple overlap: the ACR being accessbile to the client it is able to find out who has access in order to guide its decision on what credential to present (detailed argument moved to issue 145)
```

Comment:
```suggestion
Aaron: if RS needs to provide link to AS, how this would work with solid?

Pavlik: I mean AS associated with RS, distinct from IdP associated with user

Benoît: As implementation, it's sometimes hard to know which panel is responsible for which part. Clarifying the overlap would help on implementation side.

Henry: In the HttpSig approach, there is a simple overlap: the ACR being accessible by the client, it is able to find out who has access in order to guide its decision on what credential to present (detailed argument moved to issue 145)
```